jose.py

```diff
@@ -35,13 +35,15 @@
     'br?',
     'u died [real] [Not ClickBait]',
     'rEEEEEEEEEEEEE',
+    'you didn\'t buy gold',
     'not enough permissions lul',
     'you sure you can run this?',
 ]
 
 BAD_ARG_MESSAGES = [
     'dude give me the right thing',
     'u can\'t give me this and think i can do something',
+    'come to brazil',
```

Comment:
I mean, this really doesn't make sense here. lines 44, 45, 48 and 49 are descriptive, while this is not. I don't think that this belongs here.

Reply:
I agree. "come to brazil" doesn't really signify anything meaningful (LOL) and can confuse users.

Reply:
succ doesn't give much info either

Reply:
Agreed, notice how I left out line 47